src/components/Capture/capture.js

```diff
@@ -252,12 +182,7 @@ class Capture extends Component {
     this.setError('SERVER_ERROR')
   }
 
-  onGlareWarning = () => {
-    this.setWarning('GLARE_DETECTED')
-  }
-
   setError = (name) => this.setState({error: {name, type: 'error'}})
```

Comment:
This setError function could just do `this.setState({error: name})` because the concept of 'warnings' and error types has been removed from capture